plugins/secrets-manager-bitwarden/src/static/startup/providers/secrets/secretsManager.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable, Inject } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+import { SecretsManagerServiceBase } from "./base/secretsManager.service.base";
+
+@Injectable()
+export class SecretsManagerService extends SecretsManagerServiceBase {
+  constructor(
+    @Inject("BITWARDEN_SECRETS_MANAGER")
+    protected readonly secrets: Partial<Record<string, unknown>>,
+    protected readonly configService: ConfigService
+  ) {
+    super(configService);
+  }
+
+  async getSecret<T>(key: string): Promise<T | null> {
```

Comment:
@ChenemiKen with the new plugin events for secret manager, it also changed the interface for SecretsManagerService as `SecretsManagerServiceBase` expose the  method `getSecret` that accept a parameter of type Secrets.

Reviewing this PR made us aware that the AWS SecretsManager plugin suffer of the same issue and this was the fix:
https://github.com/amplication/plugins/pull/292

Sorry for the inconvenient, could you please update this PR?

Reply:
@overbit, @Shurtu-gal  I have made this update and pushed. please help review. Thanks